### DISCUSSION TITLE
GET API: /products/featured api on home screen is throwing 404

### DISCUSSION THREAD

**Discussion by mrashasahmad** [aws-samples/retail-demo-store#348] [General]
Access to XMLHttpRequest at 'http://retai-loadb-35motrukog7o-335195866.us-east-1.elb.amazonaws.com/products/featured' from origin 'http://d198z12tj1vzs0.cloudfront.net' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource.

Can anyone help? My home screen is not stopping loading and hence stuck.

**Comment by BastLeblanc** [comment]
closing as it's an old issue. Please re-open if you still encounter the issue
